packages/govern-subgraph/src/OptimisticQueue.ts

```diff
@@ -11,309 +11,320 @@ import {
+  const currentRoles = queue.roles
+  currentRoles.push(role.id)
+  queue.roles = currentRoles
 
-  queue.save();
+  queue.save()
 }
 
 export function handleRevoked(event: RevokedEvent): void {
-  const queue = loadOrCreateQueue(event.address);
+  const queue = loadOrCreateQueue(event.address)
 
-  const role = roleRevoked(event.address, event.params.role, event.params.who);
+  const role = roleRevoked(event.address, event.params.role, event.params.who)
 
   // add the role
-  const currentRoles = queue.roles;
-  currentRoles.push(role.id);
-  queue.roles = currentRoles;
+  const currentRoles = queue.roles
+  currentRoles.push(role.id)
+  queue.roles = currentRoles
 
-  queue.save();
+  queue.save()
 }
 
 // Helpers
 
 export function loadOrCreateQueue(entity: Address): OptimisticQueueEntity {
-  const queueId = entity.toHexString();
+  const queueId = entity.toHexString()
   // Create queue
-  let queue = OptimisticQueueEntity.load(queueId);
+  let queue = OptimisticQueueEntity.load(queueId)
   if (queue === null) {
-    queue = new OptimisticQueueEntity(queueId);
-    queue.address = entity;
-    queue.queue = [];
-    queue.executions = [];
-    queue.roles = [];
+    queue = new OptimisticQueueEntity(queueId)
+    queue.address = entity
+    queue.queue = []
+    queue.executions = []
+    queue.roles = []
   }
-  return queue!;
+  return queue!
 }
 
 function loadOrCreateConfig(entity: Address): ConfigEntity {
-  const configId = entity.toHexString();
+  const configId = entity.toHexString()
   // Create config
-  let config = ConfigEntity.load(configId);
+  let config = ConfigEntity.load(configId)
   if (config === null) {
-    config = new ConfigEntity(configId);
-    config.queue = entity.toHexString();
+    config = new ConfigEntity(configId)
+    config.queue = entity.toHexString()
   }
-  return config!;
+  return config!
 }
 
 function loadOrCreateCollateral(
   event: ethereum.Event,
   index: String
 ): CollateralEntity {
-  const collateralId = event.transaction.hash.toHexString() + index;
+  const collateralId = event.transaction.hash.toHexString() + index
   // Create collateral
-  let collateral = CollateralEntity.load(collateralId);
+  let collateral = CollateralEntity.load(collateralId)
   if (collateral === null) {
-    collateral = new CollateralEntity(collateralId);
+    collateral = new CollateralEntity(collateralId)
   }
-  return collateral!;
+  return collateral!
 }
 
 function loadOrCreateItem(
   containerHash: Bytes,
   event: ethereum.Event
 ): ItemEntity {
-  const itemId = containerHash.toHexString();
+  const itemId = containerHash.toHexString()
   // Create item
-  let item = ItemEntity.load(itemId);
+  let item = ItemEntity.load(itemId)
   if (item === null) {
-    item = new ItemEntity(itemId);
-    item.status = NONE_STATUS;
-    item.actions = [];
-    item.createdAt = event.block.timestamp;
+    item = new ItemEntity(itemId)
+    item.status = NONE_STATUS
+    item.actions = []
```

Comment:
I think we're not handling this `actions` array anywhere else.

Reply:
Actually we should remove this line because the relation is derived from each `Action` entity.